man/gh.Rd

```diff
@@ -29,17 +29,18 @@ API calls are used to get them, and this can take a potentially
 long time.}
 
 \item{.send_headers}{Named character vector of header field values
-  (excepting \code{Authorization}, which is handled via
-  \code{.token}). This can be used to override or augment the
-  defaults, which are as follows: the \code{Accept} field defaults
-  to \code{"application/vnd.github.v3+json"} and the
-  \code{User-Agent} field defaults to
-  \code{"https://github.com/gaborcsardi/whoami"}. This can be used
-  to, e.g., provide a custom media type, in order to access a
-  preview feature of the API.}
+(excepting \code{Authorization}, which is handled via
+\code{.token}). This can be used to override or augment the
+defaults, which are as follows: the \code{Accept} field defaults
+to \code{"application/vnd.github.v3+json"} and the
+\code{User-Agent} field defaults to
+\code{"https://github.com/gaborcsardi/whoami"}. This can be used
+to, e.g., provide a custom media type, in order to access a
+preview feature of the API.}
 }
 \value{
-Answer from the API.
+Answer from the API as a \code{gh_response} object, which is also a
+  \code{list}.
```

Comment:
Could you please remove this one, too? Seems just re-indenting. Sorry for being so picky.